docs/src/gui/pyvcp.adoc

```diff
@@ -625,32 +624,30 @@ The above code produced this example:
 image::images/pyvcp_s32.png[]
 
 .u32 Number
-
 The u32 number widget displays the value of a u32 number. The syntax
 is the same as 'number' except the name which is <u32>.
 
 .Bar
-
 A bar widget displays the value of a FLOAT signal both graphically
 using a bar display and numerically.
 The color of the bar can be set as one color throughout its range (default
 using fillcolor) or set to change color dependent upon the value of the halpin
 (range1, range2 range3 must all be set, if you only want 2 ranges, set 2 of
 them to the same color).
 
-* <halpin>"my-bar"</halpin> text, sets the pin name, pyvcp.my-bar
-* <min_>0</min_> number, sets the minimum scale
-* <max_>140</max_> number, sets the maximum scale
-* <format>"3.1f"</format> text, sets the number format using Python number formatting
-* <bgcolor>"grey"</bgcolor> text, sets the background color
-* <fillcolor>"red"</fillcolor> text, sets the fill color
-* <range1>0,100,"green"</range1> number, number, text, sets the first range and color
-* <range2>101,135,"orange"</range2> number, number, text, sets the first range and color
-* <range3>136, 150,"red"</range3> number, number, text, sets the first range and color
-* <canvas_width>200</canvas_width> number, sets the overall width
-* <canvas_height>50</canvas_height> number, sets the overall height
-* <bar_height>30</bar_height> number, sets the bar height, must be less than canvas_height
-* <bar_width>150</bar_width> number, sets the bar width, must be less than canvas_width
+* `<halpin>"my-bar"</halpin>` (text), derives and sets the pin name: `pyvcp.my-bar`.
```

Comment:
And here they are suddenly mono-spaced. Inconsistent.

Reply:
monospace for pins is correct imho since it is something that is communicated as part of the programming.

Reply:
In the most occurrences in this document they are italic, that I wanted to express. Either change them to monospace or these to italic. But consistent.

Reply:
Let's fix this with next pass...